src/provider/handler/mappers/postMapper.js

```diff
@@ -3,16 +3,15 @@ export function mapPost(post) {
     type: {
       value: 'article'
     },
+    id: post.id,
+    title: (post.title && post.title.rendered) ? post.title.rendered : '',
```

Comment:
same here, you may want to reuse this logic somewhere else - you could extract a `getPostFullTitle` function or something